plugins/cue_points/quiz/QuizPlugin.php

```diff
@@ -973,7 +973,245 @@ private static function getQuizElasticSearchData($entry)
 
 		return $quizData;
 	}
-
+	
+	/* (non-PHPdoc)
+	 * @see IKalturaCuePointXmlParser::parseXml()
+	 */
+	public static function parseXml(SimpleXMLElement $scene, $partnerId, CuePoint $cuePoint = null)
+	{
+		$sceneName = $scene->getName();
+		
+		switch ($sceneName)
+		{
+			case "scene-question-cue-point":
```

Comment:
This case will succeed if the $sceneName will contain other values.
Must use either === or strcmp
See - 
 function a($v) { switch($v) { case "11212" : print 'ok'; break;};
Moshe Maor
    a(true);
Will print 'ok'